### PR TITLE
Connected postgres to main database

### PR DESCRIPTION
**Pull request by Benntobox**
Postgres database now properly connected to server

### CODE REVIEW

**Review by sunnymakin — APPROVED** [review]
🎉 🎉 🎉 

- Great work on code readability and naming conventions!!!
- DB scripts look good, easy to understand!

File: database/postgresdb.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const { Pool } = require('pg');
+const db = require('./seed-data/postgres.js');
+
+const client = new Pool({
```
Comment (jhou12) [inline database/postgresdb.js:4]: def use env vars when deployed!

File: database/seed-data/postgres.js [hunk 2]
```diff
@@ -40,24 +29,19 @@ let adder = async (filecount) => {
...
 }
 
-let bigtester = async () => {
-  console.log('POWERING UP')
-  let data = await client.query(`EXPLAIN ANALYZE SELECT * FROM features WHERE productid = 2000000;`);
-  let result = await client.query(`SELECT * FROM features WHERE productid = 2000000;`);
-  console.log(data.rows);
-  console.log(result.rows);
-};
+let analyze = async(client, productid) => {
```
Comment (jhou12) [inline database/seed-data/postgres.js:40]: good job on verifying!

**Review by jhou12 — COMMENTED** [review]
@Benntobox good job connecting the db, creating fallback data, and verifying! sorry for the late review :)

**Pull request merged**